starlette/_crypto.py

```diff
@@ -0,0 +1,14 @@
+import hashlib
+import inspect
+
+
+# TODO: Remove when dropping support for PY38. inspect.signature() is used to
+# detect whether the usedforsecurity argument is available as this fix may also
+# have been applied by downstream package maintainers to other versions in
+# their repositories.
+if "usedforsecurity" in inspect.signature(hashlib.md5).parameters:
+    def md5_hexdigest(data: bytes, *, usedforsecurity: bool = True) -> str:
+        return hashlib.md5(data, usedforsecurity=usedforsecurity).hexdigest()
```

Comment:
I decided to return the hexdigest directly because:
1. That's the only way Starlette is currently using it
2. It's easier to type the signature
3. It's a private implementation so we can easily change it in the future

Reply:
It's fine.